CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+-----------------------------
+* Add an option to scale the number of instances via
+  `HASTEXO_REPLICA_COUNT` setting.
```

Comment:
I think you should rename this variable, and make a variable of its kind available not for to the `hastexo-xblock` deployment. We'll also want to modify the replica count for the reaper and suspender deployments, in order to switch them on and off.

Or, else, if you determine that the reaper and suspender should only ever run a single Pod or none at all, you could also create boolean variables (like `HASTEXO_ENABLE_REAPER` and `HASTEXO_ENABLE_SUSPENDER`), and based on that set 0 or 1 replicas for them. (Just removing the configuration when those booleans are unset won't cut it.)

Reply:
Well, I don't know how could I determine how many reaper and suspender Pods should ever run, so I guess a general setting for the count should be a good option. Do you have a specific naming convention in mind? Should it be then `HASTEXO_COUNT`, HASTEXO_REAPER_COUNT` and `HASTEXO_SUSPENDER_COUNT`?

Reply:
On second thought it doesn't make sense to ever have more than 1 replica for the suspender and reaper, so booleans are more appropriate for that. That means that this variable name (and the patch) can stay as-is, and we can handle the additional boolean variables in a separate commit.